api/src/main/java/org/openmrs/module/mirebalaisreports/definitions/FullDataExportBuilder.java

```diff
@@ -151,6 +151,10 @@
                 configurations.add(new Configuration(MirebalaisReportsProperties.ED_TRIAGE_DATA_EXPORT_REPORT_DEFINITION_UUID, "edtriagedataexport",
                         Arrays.asList("edTriage")));
             }
+            if (config.isComponentEnabled(Components.SOCIAL_ECONOMICS) || config.isComponentEnabled(Components.ALL_DATA_EXPORTS)) {
```

Comment:
I don't think we need a new component for this (and this is probably why the report was't showing up... you'd need to add this component to this list of components in pih-config-mirebaiais*. json files.

I think we just want to add this any time we are in a Haiti environment, so instead of creating a new if/then block we should just include lines 155-156 below in the if/then block found at lines 77-80

Reply:
Thanks Mark, I worked on this.